fail validations on non-ignorable error codes (#2044)
demisto/demisto-sdk: CHANGELOG.md

# Changelog

## Unreleased

> * Added support in the **validate** command to fail validations on non-ignorable errors which are part of the .pack-ignore.

Reviewer: ```suggestion
* Added a validation to the **validate** command, failing when non-ignorable errors are present in  .pack-ignore.
```
